src/test/unit/rcdevice_unittest.cc.txt

```diff
@@ -796,41 +780,41 @@ extern "C" {
             return buffer[testData.responseDataReadPos++];
         }
 
-        return 0;
+        return 0; 
     }
 
-    void sbufWriteString(sbuf_t *dst, const char *string)
-    {
-        UNUSED(dst); UNUSED(string);
+    void sbufWriteString(sbuf_t *dst, const char *string) 
+    { 
```

Comment:
Trailing whitespace in this line, previous one and next one. Also, use just one `UNUSED()` per line.